packages/orchestration/test/exos/local-chain-account-kit.test.ts

```diff
@@ -9,6 +9,74 @@ import { prepareMockChainInfo } from '../../src/utils/mockChainInfo.js';
 import { NANOSECONDS_PER_SECOND } from '../../src/utils/time.js';
 import { commonSetup } from '../supports.js';
 
+test('deposit, withdraw', async t => {
+  const { bootstrap, brands, utils } = await commonSetup(t);
+
+  const { bld: stake } = brands;
+
+  const { timer, localchain, marshaller, rootZone, storage } = bootstrap;
```

Comment:
Returning `localchain` as part of `bootstrap` reads a little funny.

Reply:
because the localchain vat isn't available at bootstrap? This object is an ersatz version of the bootstrap vat's "space", not necessarily something made during bootstrap (which contracts shouldn't be concerned with)

would it help to rename this to `spacey` or something like that?

Reply:
maybe `coreSpace` or `coreEvalSpace`?

Strictly speaking, `marshaller` isn't in there - only `board`. And `rootZone` is called `zone`. But I'm OK reading past / around those.

I'm not sure I get the point about contracts.

On reflection, I think the bottom line is: the reader just has to know (or read the source to confirm) that `t.context` here is a state after `localchain` has been deployed. I'm not sure any particular rename is going to help much.